tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js

```diff
@@ -266,16 +284,11 @@ test.describe( 'Complete your campaign', () => {
 							textContent
 						);
 
-					const responsePromise =
```

Comment:
Why do we no longer need to wait for this response after the country setting is changed? It looks like when the country values change, the text that reads "Tip: Most merchants targeting similar countries set a daily budget of %d USD" does change, so we should probably ensure that is still being tested.

Reply:
@ankitrox I'm still unsure why we need to remove this. Could you clarify?

Reply:
@ankitrox Can you check @joemcgill 's comment please? Not sure about this one.